unit-tests/run-unit-tests.py

```diff
@@ -541,7 +552,15 @@ def test_wrapper( test, configuration = None ):
         test.debug_dump()
         #
         if tag and tag not in test.config.tags:
-            log.d( 'does not fit --tag:', test.tags )
+            log.d( 'does not fit --tag:', test.config.tags )
+            continue
+        #
+        if list_tags:
+            tags.update( test.config.tags )
+            continue

```

Comment:
So if we list tags we can't list tests?
I suggest we always update tags and tests, and only conditionally display them after we're done.

Reply:
Done